backend/audit/etl.py

```diff
@@ -333,76 +340,98 @@ def load_general(self):
         general.save()
 
     def load_secondary_auditor(self):
+        general = self._get_general()
+        if not general:
+            return
         secondary_auditors = self.single_audit_checklist.secondary_auditors
-        # MCJ This might be empty
-        if secondary_auditors:
-            if "secondary_auditors_entries" in secondary_auditors["SecondaryAuditors"]:
-                for secondary_auditor in secondary_auditors["SecondaryAuditors"][
-                    "secondary_auditors_entries"
-                ]:
-                    sec_auditor = SecondaryAuditor(
-                        report_id=self.single_audit_checklist.report_id,
-                        auditor_ein=secondary_auditor["secondary_auditor_ein"],
-                        auditor_name=secondary_auditor["secondary_auditor_name"],
-                        contact_name=secondary_auditor[
-                            "secondary_auditor_contact_name"
-                        ],
-                        contact_title=secondary_auditor[
-                            "secondary_auditor_contact_title"
-                        ],
-                        contact_email=secondary_auditor[
-                            "secondary_auditor_contact_email"
-                        ],
-                        contact_phone=secondary_auditor[
-                            "secondary_auditor_contact_phone"
-                        ],
-                        address_street=secondary_auditor[
-                            "secondary_auditor_address_street"
-                        ],
-                        address_city=secondary_auditor[
-                            "secondary_auditor_address_city"
-                        ],
-                        address_state=secondary_auditor[
-                            "secondary_auditor_address_state"
-                        ],
-                        address_zipcode=secondary_auditor[
-                            "secondary_auditor_address_zipcode"
-                        ],
-                    )
-                    sec_auditor.save()
+        if not secondary_auditors:
+            general.secondary_auditors_exist = False
+            general.save
```

Comment:
```suggestion
            general.save()
```
I assume this should be a method invocation.

Reply:
This line of code has been removed.